src/forms/inputs/textarea.js

```diff
@@ -7,14 +7,16 @@ import { compose, filterInvalidDOMProps, generateInputErrorId } from '../../util
 
 /**
  *
- * A textarea input that can be used in a `redux-forms`-controlled form. Optionally displays a character count.
+ * A textarea input that can be used in a `redux-forms`-controlled form.
+ * Forwards ref down to textarea input and optionally displays a character count.
  *
  * @name Textarea
  * @type Function
  * @param {Object} input - A `redux-forms` [input](http://redux-form.com/6.5.0/docs/api/Field.md/#input-props) object
  * @param {Object} meta - A `redux-forms` [meta](http://redux-form.com/6.5.0/docs/api/Field.md/#meta-props) object
  * @param {Number} [maxLength] - The maximum allowed length of the input
  * @param {Boolean} [hideCharacterCount=false] - Whether to hide the character count if given a maxLength
+ * @param {Ref} [forwardedRef] - A ref to be forwarded to `textarea` input when used with `redux-forms`
```

Comment:
Perhaps I'm misunderstanding but since we are using `forwardRef` imported from `React`, is using this prop actually coupled to `redux-forms`?

Reply:
We're adding this to help support usage in redux-forms, but technically it will always accept a `forwardedRef` prop, regardless of the form library. I think I'd prefer for this comment to be agnostic of the form library.

Reply:
👍 I attempted to explain more generally why you would use `forwardedRef`.